GameOfLife/View/GameView.cs

```diff
@@ -9,17 +9,17 @@ public class GameView
         /// Makes view of the game field.
         /// </summary>
         /// <param name="args">Array containing the game field.</param>
-        public void View(int[,] args)
+        public static void View(int[,] args)
         {
             Console.Clear();
             for (int row = 0; row < args.GetLength(1); row++)
             {
                 for (int column = 0; column < args.GetLength(0); column++)
                 {
-                    Console.Write(args[column , row] == 1 ? "*" : " ");
+                    Console.Write(args[column, row] == 1 ? "*" : " ");
                 }
                 Console.WriteLine();
```

Comment:
Always insert empty line after closing } if that is not followed  by closing } of inclosing block

Reply:
Empty lines added.